misc/python/materialize/cli/mzexplore.py

```diff
@@ -296,6 +356,68 @@ def changes(
         raise click.ClickException(msg) from e
 
 
+@app.group()
```

Comment:
Are there other things we'll clone later (as opposed to just writing `mzexplore clone-defs`)?

Reply:
Not sure, I'm following the convention established with the other modules where the Python module name is the verb (`extract`, `analyze`, `clone`) and the function name is the subject.